Examples/php/class/runme.php

```diff
@@ -14,7 +14,7 @@
 
 # ----- Access a static member -----
 
-print "\nA total of " . Shape::nshapes() . " shapes were created\n";
+print "\nA total of " . Shape::nshapes_get() . " shapes were created\n";
 
```

Comment:
Ideally the new-style wrapping should provide the same interface in PHP, and we can avoid such changes to the examples, etc.